applications/app/views/fragments/crosswords/crosswordMetaHeader.scala.html

```diff
@@ -37,7 +37,11 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(crosswordPage.c
                     <div class="meta__save-for-later js-save-for-later" data-position="top"></div>
                 }
                 <div class="meta__numbers modern-visible">
-                    <div class="u-h meta__number" data-discussion-id="@crosswordPage.content.content.discussionId" data-commentcount-format="content" data-discussion-closed="@{crosswordPage.content.trail.isCommentable}">
+                    <div class="meta__number">
+                        <a href="#comments" data-link-name="Comment count" class="commentcount2 tone-colour">
+                            <h3 class="commentcount2__heading">@fragments.inlineSvg("comment-16", "icon", List("inline-tone-fill")) Comments</h3>
```

Comment:
Feels a bit unjustified to use a heading here. This is just a number after all, it's not really worth appearing in the document outline (and the accessibility tree, with potentially dozens of identical "Comments", will only slow down people browsing by heading)